src/core/model.js

```diff
@@ -455,4 +458,17 @@ export default class Model extends Object3D {
   setTextures(textures = []) {
     throw new Error('model.setTextures replaced: setUniforms({sampler2D: new Texture2D})');
   }
+
+  isPickable() {
```

Comment:
Since picking is not generally implemented in luma.gl, do we still want to have this property here